web/cmd/directremotecontrol/main.go

```diff
@@ -53,7 +53,7 @@ type Arguments struct {
 	Port   utils.NetPortFlag `flag:"port,default=8080"`
 }
 
-var logger = logging.NewDebugLogger("rdk")
+var logger = logging.NewDebugLogger("entrypoint")
```

Comment:
Should we call this one `direct-remote-control`? I don't think we ever reach the entrypoint files from here.

Reply:
yeah that's better, I don't like that they were all the same anyway